src/components/Navbar/index.spec.tsx

```diff
@@ -513,16 +431,7 @@ describe(__filename, () => {
     }) => {
       root.setProps({
         dispatch: jest.fn(),
-        ...mapStateToProps(store.getState(), {
-          history: createFakeHistory(),
-          location: createFakeLocation(),
-          match: {
-            params: {},
-            isExact: true,
-            path: '/some-path',
-            url: 'some-url',
-          },
-        }),
+        ...mapStateToProps(store.getState()),
```

Comment:
Why is this change needed?

Reply:
Because `ownProps` was removed as an argument to `mapStateToProps`.